### ISSUE TITLE
Consider updating to pylibjpeg-libjpeg 1.3.2 (or pylibjpeg[jpeg] 1.4)  and deprecate pymedphys-pylibjpeg-libjpeg (address arm64 install issue)

### ISSUE THREAD

**Issue by sjswerdloff** [pymedphys/pymedphys#1736]
Attempting to install pymedphys-pylibjpeg-libjpeg 1.3.1 has the same problem as pylibjpeg-libjpeg on Arm64 (Apple M1 and at least Ubuntu 22.04 VMs running on an Apple M1).
There is a pylibjpeg-libjpeg 1.4.
Changing the pyproject.toml to comment out the pymedphys flavour and uncommenting the parent version seems to work

**Comment by SimonBiggs** [comment]
Yup, happy for that change. The pymedphys variant was just a temporary option waiting for the main package to upgrade and provide wheels.

It just needs to be undergone in the pyproject.toml and then poetry update needs to be run.

**Comment by SimonBiggs** [comment]
@pchlap, could we make sure within the release process that we change `pymedphys-pylibjpeg-libjpeg` back to `pylibjpeg-libjpeg`

**Comment by SimonBiggs** [comment]
> There is a pylibjpeg-libjpeg 1.4.

The latest version online appears to still be 1.3.1:
https://pypi.org/project/pylibjpeg-libjpeg/

Therefore still broken with:
https://github.com/pymedphys/pymedphys/issues/1556#issuecomment-1152821285

**Comment by SimonBiggs** [comment]
Blocked by:
https://github.com/pydicom/pylibjpeg-libjpeg/issues/39 and https://github.com/pydicom/pylibjpeg-libjpeg/issues/38

**Comment by sjswerdloff** [comment]
https://pypi.org/project/pylibjpeg/

**Comment by sjswerdloff** [comment]
So... there is a pylibjpeg 1.4
But not a pylibjpeg-libjpeg 1.4
When I did a local substitute of pylibjpeg 1.4 for pylibjpeg-libjpeg 1.3.1, it worked.

**Comment by sjswerdloff** [comment]
I mangled the names of things a bit in the issue and my comment.

**Comment by SimonBiggs** [comment]
A new pylibjpeg release has been created which fixes the M1 issues:

https://github.com/pydicom/pylibjpeg-libjpeg/issues/39 

@sjswerdloff, would you be open to forging a `0.39.1` release following @pchlap's document? The only release item for `0.39.1` would be swapping back to [pylibjpeg-libjpeg](https://github.com/pydicom/pylibjpeg-libjpeg) with a lower pin of `1.3.2`?

**Comment by sjswerdloff** [comment]
> A new pylibjpeg release has been created which fixes the M1 issues:
> 
> [pydicom/pylibjpeg-libjpeg#39](https://github.com/pydicom/pylibjpeg-libjpeg/issues/39)
> 
> @sjswerdloff, would you be open to forging a `0.39.1` release following @pchlap's document? The only release item for `0.39.1` would be swapping back to [pylibjpeg-libjpeg](https://github.com/pydicom/pylibjpeg-libjpeg) with a lower pin of `1.3.2`?

I'll give it a go, but probably not until this coming weekend

**Comment by sjswerdloff** [comment]
I don't see a 0.39(.0) branch

**Comment by sjswerdloff** [comment]
Not exactly a catch-22, but the documentation of how to do a release is in main, but not part of 0.39.x (v0.39.0 tag).
The instructions appear quite accurate and complete for doing a feature release, but not quite complete for a patch release.
That's not a gripe.  This kind of makes the point that writing the instructions for a feature release while doing a feature release makes sense, and that updating the instructions to address a patch release while doing a patch release also makes sense...

I assume that formally a patch is done against the parent branch rather than main.
I've checked out the tag for v0.39.0 to a branch (0.39.x) and pushed upstream.
I think either the instructions or the automation need to address having the Major.Minor.x branch created, possibly just after the tag for the feature release.

I've made changes relative to that branch and pushed *that* upstream.
Doing a bug fix release is slightly different than what was documented.
I've followed the instructions (that are available from the main branch), but did it from the parent/release branch, and then created the draft PR between the 0.39.1-release-prep branch (per the instructions) and the 0.39.x branch (not discussed in the instructions, and created by me just in time based on the tag).

So a few questions:  
Should I merge the release instructions in to the bug fix release, and update them to address bug fix releases (and also update main)?
Or just do the minimum for the bug fix release, and put the updates to the release instructions in to main, and wait for the next feature release for those to show up?

**Comment by pchlap** [comment]
Hi @sjswerdloff, ah yes I had completely omitted the part around creating the 0.39.x branch. Thank you for rectifying that for me, apologies that just somehow wasn't at all on my radar.

I contributed the guidance doc after having completed the release. That's because I was writing the doc as working through the process so wanted to have a completed doc to contribute. I also assumed there might be some additional work to do on the doc and I didn't want to hold up the release in case that took some time.

> Or just do the minimum for the bug fix release, and put the updates to the release instructions in to main, and wait for the next feature release for those to show up?

I'd suggest this approach would be fine here. Since there is still a small group of us working through the release process I expect we won't be missing this portion of the docs until the next feature release.

**Comment by sjswerdloff** [comment]
@pchlap , it looks like @Matthew-Jennings has done some updates to the release instructions.
I am working on a PR that just tries to update  the references involving pylibjpeg-libjpeg (vs tweaked variant pymedphys-pylibjpeg-libjpeg) in main.
We could have another PR that just focuses on updating the release instructions
